src/client-entry.js

```diff
@@ -208,3 +208,6 @@ function checkiIsOnline () {
 window.addEventListener('online', checkiIsOnline)
 window.addEventListener('offline', checkiIsOnline)
 
+EventBus.$on('user.loggedin', (receivedData) => {
```

Comment:
Please change the event name to be compliant with the convention: module-after|before-eventname so in that case it should be: user-after-loggedin